src/js/components/addData/AddDataHeader.jsx

```diff
@@ -60,10 +60,14 @@ export default class AddDataHeader extends React.Component {
     }
 
     render() {
-        let lastUpdated = null;
-        if (this.state.ready && this.state.last_updated){
+        let submissionContext = null;
+        if (this.state.ready) {
             let formattedTime = moment.utc(this.state.last_updated).local().format('h:mm a');
-            lastUpdated = <LastUpdated last_updated={formattedTime} />
+            submissionContext = <SubmissionContext
+              formattedTime={formattedTime}
+              agencyName={this.state.agency_name}
+              submissionTime={this.state.reporting_period_start_date}
```

Comment:
A user can also select a specific month instead of a quarter for a fiscal year, but we're using the `reporting_period_start_date` to convert it to Quarter/FY format right? Do we need to account for the selection of a specific month?